frontend/packages/dev-console/console-extensions.json

```diff
@@ -186,6 +186,21 @@
       ]
     }
   },
+  {
+    "type": "dev-console.import/environment",
+    "properties": {
+      "imageStreamName": "nodejs",
+      "imageStreamTags": ["14-ubi8", "14-ubi8-minimal", "12-ubi8", "latest"],
```

Comment:
@rottencandy @christianvogt Any thoughts here if we should also add node `16-` or `ubi9` here already or should we update this shortly before we release 4.10? ... I will create a story so that we can check in 2-3 month which versions we should support.

See https://github.com/sclorg/s2i-nodejs-container/pull/295

Reply:
Don't think it would affect anything so added `16-ubi8` to the list.